src/lib389/cli/dscontainer

```diff
@@ -89,14 +90,17 @@ def _begin_environment_config():
     # TODO: Should we set replication agreements from env?
     autotune_pct = os.getenv("DS_MEMORY_PERCENTAGE", None)
     if autotune_pct is not None:
-        try:
-            autotune_pct = int(autotune_pct)
-        except:
-            log.error("Invalid DS_MEMORY_PERCENTAGE - resetting to system default value")
-            autotune_pct = 0
-        log.debug("Setting LDBM Autotune Percentage to: %s", autotune_pct)
-        ldbmconfig = LDBMConfig(inst)
-        ldbmconfig.set("nsslapd-cache-autosize", str(autotune_pct))
+        if get_default_db_lib() == "mdb":
+            log.error("DS_MEMORY_PERCENTAGE is present, but setting entry cache is not supported for MDB")
+        else:
+            try:
+                autotune_pct = int(autotune_pct)
+            except:
+                log.error("Invalid DS_MEMORY_PERCENTAGE - resetting to system default value")
```

Comment:
I'm not sure if this should be an error or warn, feedback is welcome.

Reply:
It's not technically breaking anything, so I think it should be a warning

Reply:
Agreed, should be a warning.